proxies/live/apiproxy/policies/AssignMessage.Swap.nhsd-ers-obo-user-id.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<AssignMessage async="false" continueOnError="false" enabled="true" name="AssignMessage.Swap.nhsd-ers-obo-user-id">
+    <Set>
+        <Headers>
+            <Header name="x-ers-obo-user-id">{message.header.nhsd-ers-obo-user-id}</Header>
+        </Headers>
```

Comment:
@nhsd-nigel-mellor  tests have now been added to test the headers. You could easily add these headers to the test and that will ensure the values are copied over.